src/Microsoft.Framework.PackageManager/Bundle/BundleOperations.cs

```diff
@@ -17,15 +18,40 @@ public void Delete(string folderPath)
             FileOperationUtils.DeleteFolder(folderPath);
         }
 
-        public void Copy(string sourcePath, string targetPath)
+        public void Copy(IEnumerable<string> sourceFiles, string sourceDirectory, string targetDirectory)
         {
-            Copy(
-                sourcePath, 
-                targetPath, 
-                shouldInclude: _ => true);
-        }
+            sourceDirectory = PathUtility.EnsureTrailingSlash(sourceDirectory);
+            targetDirectory = PathUtility.EnsureTrailingSlash(targetDirectory);
+
+            foreach (var sourceFilePath in sourceFiles)
+            {
+                var fileName = Path.GetFileName(sourceFilePath);
+                Debug.Assert(fileName != null, "fileName != null");
```

Comment:
In addition to null value check, I guess you also want to check the case where `sourceFilePath` is a path ending with path separator (i.e. a path to dir). In this case, `fileName` is `string.Empty`.

So I suggest use `string.IsNullOrEmpty(fileName)`